examples/compound/README.md

```diff
@@ -1,2 +1,288 @@
 # Compound Connector
+
 This is an example compound connector that uses `EthereumConnector` as a base connector.
+
+# How to Build a Connector
+
+## Prerequisites
+
+1. Golang 1.18+
+2. Docker
+3. protoc
+4. protoc-gen-go
+5. abigen
+6. solc
+
+## Overview
+
+We will be building a source connector. As a reminder,a source connector extracts/ingests, cleans, and generalizes data
+from an external source, usually a blockchain, smart contract, or API. It then passes on this data to the Nakji message
+queue. **It is highly recommended to use the example compound connector as the template to build your own connector.**
```

Comment:
>compound connector

can you rephrase it to something like `connector built for Compound lending platform`?

Reply:
Will refine the whole docs later.